_pages/docs/matching-options.md

```diff
@@ -74,6 +74,90 @@ Here's an example of match configuration options.
 </options>
 ```
 
+And here are the same options in JSON format:
+
+```json
+{
+  "options": {
+    "property-defs": {
+      "property": [
+        { "namespace": "", "localname": "IdentificationID", "name": "ssn" },
+        { "namespace": "", "localname": "PersonGivenName", "name": "first-name" },
+        { "namespace": "", "localname": "PersonSurName", "name": "last-name" },
+        { "namespace": "", "localname": "AddressPrivateMailboxText", "name": "addr1" },
+        { "namespace": "", "localname": "LocationCity", "name": "city" },
+        { "namespace": "", "localname": "LocationState", "name": "state" },
+        { "namespace": "", "localname": "LocationPostalCode", "name": "zip" }
+      ]
+    },
+    "algorithms": {
+      "algorithm": [
```

Comment:
this feels lame. I'd expect algorithms: []

Reply:
for merging, the algorithms object has keys `custom` (array) and `stdAlgorithm`. I'm inclined to leave `algorithms` as an object and to change `algorithm` to `custom` to be consistent. Thoughts?

Reply:
custom works for me